### PR TITLE
Lua morse

### PR DESCRIPTION
**Pull request by d3vil0p3r**
**Fixes issue:** #1744

**Changes:**
Following changes are made in this commit:
- Implemented the Morse Code translator in Lua
- Fixed the **spacing issue** in C++ code `code/cryptography/morse_cipher/morse_code_generator.cpp`

### CODE REVIEW

File: code/cryptography/morse_cipher/morse_code_generator.cpp [hunk 1]
```diff
@@ -8,32 +8,32 @@ void encryption(string msg)
...
 encrypt['5']="....."; encrypt['6']="-...."; encrypt['7']="--...";
-encrypt['8']="---.."; encrypt['9']="----."; encrypt['0']="-----"; 
+encrypt['8']="---.."; encrypt['9']="----."; encrypt['0']="-----";
 
 string cipher;
 for(int i=0;i<msg.size();i++)
 {
   	if(msg[i]!=' ')
-   	cipher = cipher + encrypt[msg[i]];
+   	cipher = cipher + encrypt[msg[i]] + " ";
```
Comment (arnavb) [inline code/cryptography/morse_cipher/morse_code_generator.cpp]: Fix the indentation here and below.

File: code/cryptography/morse_cipher/morse_code_generator.cpp [hunk 2]
```diff
@@ -8,32 +8,32 @@ void encryption(string msg)
...
    	else
-   	cipher = cipher + " ";
+   	cipher = cipher + "  ";
 }
```
Comment (arnavb) [inline code/cryptography/morse_cipher/morse_code_generator.cpp]: I think this brace needs to be moved to the right.

File: code/cryptography/morse_cipher/morse_code_generator.cpp [hunk 3]
```diff
@@ -8,32 +8,32 @@ void encryption(string msg)
...
 	cout<<cipher;
 }
 //Driver program
 int main()
 {
 	string key="CONVERT THIS MORSE CODE";
```
Comment (arnavb) [inline code/cryptography/morse_cipher/morse_code_generator.cpp]: CONVERT THIS _TO_ MORSE CODE

**Comment by d3vil0p3r** [comment]
@arnavb thank you for the review. Fixes done!

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Comment by d3vil0p3r** [comment]
@AdiChat indeed enjoying, and this is just the starting 😉

**Pull request merged**